src/metrics/metrics.jl

```diff
@@ -587,16 +430,151 @@ shelter volume (a 3D metric).
+    _inputs::YAXArray = DataCube(Matrix(inputs); scenarios=1:size(inputs, 1), factors=names(inputs))
+    return _absolute_shelter_volume(X, k_area, _inputs)
+end
+function _absolute_shelter_volume(
+    X::YAXArray{T,3},
+    k_area::Vector{T},
+    inputs::YAXArray
+)::AbstractArray{T} where {T<:Real}
+    # Collate for a single scenario
+    nspecies::Int64 = size(X, :species)
+
+    # Calculate shelter volume of groups and size classes and multiply with area covered
+    ASV::YAXArray = DataCube(zeros(size(X)...), (:timesteps, :species, :sites))
+    colony_vol, _ = _colony_Lcm2_to_m3m2(inputs)
+    _shelter_species_loop!(X, ASV, nspecies, colony_vol, k_area)
+
+    # Sum over groups and size classes to estimate total shelter volume per site
+    return dropdims(sum(ASV, dims=:species), dims=:species)
+end
+function _absolute_shelter_volume(
+    X::YAXArray{T,4},
+    k_area::Vector{T},
+    inputs::YAXArray
+)::AbstractArray{T} where {T<:Real}
+    nspecies::Int64 = size(X, :species)
+
+    # Calculate shelter volume of groups and size classes and multiply with area covered
+    nscens::Int64 = size(X, :scenarios)
+    ASV = ASV::YAXArray = DataCube(zeros(size(X)...), (:timesteps, :species, :sites, :scenarios))
+    for scen::Int64 in 1:nscens
+        colony_vol, _ = _colony_Lcm2_to_m3m2(inputs[scen, :])
+        _shelter_species_loop!(X[scenarios=scen], ASV, nspecies, colony_vol, k_area)
+    end
+
+    # Sum over groups and size classes to estimate total shelter volume per site
+    return dropdims(sum(ASV, dims=:species), dims=:species)
+end
+function _absolute_shelter_volume(rs::ResultSet)::AbstractArray
+    return rs.outcomes[:absolute_shelter_volume]
+end
 absolute_shelter_volume = Metric(_absolute_shelter_volume, (:timesteps, :sites, :scenarios))
 
+"""
+_relative_shelter_volume(X::AbstractArray{T,3}, k_area::Vector{T}, inputs::DataFrame)::AbstractArray{T} where {T<:Real}
+_relative_shelter_volume(X::AbstractArray{T,3}, k_area::Vector{T}, inputs::DataFrameRow)::AbstractArray{T} where {T<:Real}
+_relative_shelter_volume(X::AbstractArray{T,4}, k_area::Vector{T}, inputs::DataFrame)::AbstractArray{T} where {T<:Real}
+_relative_shelter_volume(X::AbstractArray{T,4}, k_area::Vector{T}, inputs::DataFrameRow)::AbstractArray{T} where {T<:Real}
+_relative_shelter_volume(X::AbstractArray{T,3}, k_area::Vector{T}, inputs::YAXArray)::AbstractArray{T} where {T<:Real}
+_relative_shelter_volume(X::AbstractArray{T,4}, k_area::Vector{T}, inputs::YAXArray)::AbstractArray{T} where {T<:Real}
+relative_shelter_volume(rs::ResultSet)
+
+Provide indication of shelter volume relative to theoretical maximum volume for
+    the area covered by coral.
+
+    The metric applies log-log linear models developed by Urbina-Barreto et al., [1]
+    which uses colony diameter and planar area (2D metrics) to estimate
+    shelter volume (a 3D metric).
+
+    ```math
+    RSV = \\begin{cases}
+    TASV / MSV & TASV > 0, \\\\
+0 & \\text{otherwise}
+\\end{cases}
+```
```

Comment:
```suggestion
    ```math
    RSV = \\begin{cases}
    TASV / MSV & TASV > 0, \\\\
    0 & \\text{otherwise}
    \\end{cases}
    ```
```
Possible indent?